dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -65,6 +67,7 @@
     private final AtomicInteger queueSizeEstimate = new AtomicInteger(0);
     private final int maxQueueSize;
     private Counter queueSizeCounter;
+    private Timer queuedTime;
```

Comment:
why would this not be final?

Reply:
doh, this and the counter should be final